next.config.js

```diff
@@ -114,14 +113,15 @@ const SECURITY_HEADERS = [
 /** @type {import('next').NextConfig} */
 const nextConfig = removeImports({
   experimental: {
-    esmExternals: true,
```

Comment:
@wraeth-eth do u know what this was for? removed it and seems okay...

Reply:
NFI